kuma/wiki/tests/test_models.py

```diff
@@ -150,6 +150,14 @@ def test_document_has_legacy_namespace(slug, legacy):
     assert Document(slug=slug).has_legacy_namespace == legacy
 
 
+@pytest.mark.parametrize(
+    "slug,should_noindex",
+    [("MDN/Doc_status", True), ("MDN/Doc_status/API", True), ("Web/CSS", False)],
```

Comment:
What should `MDN/Doc_Status` return? (Case sensitivity)

What about something like `MDN/Doc_status_dashboard` (Should we only match at path boundaries, or should it be any arbitrary string prefix?)

Reply:
It's fine to be case sensitive. There are no `Doc_StatUsEs`. 
There does exist a slug called `MDN/Doc_status_clone` but I'm confidently fine with that getting excluded too.

Reply:
Okay, that's fine with me. I find case sensitivity a bit surprising here, so I'd really appreciate two changes:

1. Add a test here to demonstrate case sensitivity, `("mdn/doc_status", False)` or similar.
2. Add a comment where we define `NOINDEX_SLUG_STARTS` noting that it's case sensitive. Just a `# case sensitive` at the end of the line would be enough.

Otherwise, 👍

Reply:
Done.

Reply:
I'll merge this when it goes green now.